GUIDELINES.md

```diff
@@ -1,5 +1,25 @@
 # Engineering Guidelines
 
+## Setup
+
+1. Clone this repository.
+2. Install [Docker].
```

Comment:
Tip: you can put always `1.` and it will render proper numbers in markdown. Check the [docs](https://www.markdownguide.org/basic-syntax/#ordered-lists).